go/libraries/doltcore/remotesrv/http.go

```diff
@@ -100,16 +100,15 @@ func (fh filehandler) ServeHTTP(respWr http.ResponseWriter, req *http.Request) {
 			return
 		}
 
-		tokens := strings.Split(path, "/")
-		if len(tokens) != 3 {
+		i := strings.LastIndex(path, "/")
+		if i < 0 || len(path[i:]) != 33 {
```

Comment:
`len("https://doltremoteapi.dolthub.com")`?

Reply:
Good point, deserves a comment. `len("/" + tableFileID)`, which is also len of a serialized dolt commit hash.